demisto_sdk/commands/setup_env/setup_environment.py

```diff
@@ -455,9 +614,11 @@ def configure_integration(
         test_module (bool): Whether run test module on the instance on XSOAR/XSIAM
 
     Raises:
-        RuntimeError: If configuring failed (for instance, Docker is turned off)
+        RuntimeError: If using auto-detection for IDE and it failed,
+            or if the configuration failed (for instance, Docker is turned off)
     """
-    ide_folder = CONTENT_PATH / IDE_TO_FOLDER[ide]
+    base_path = CONTENT_PATH
```

Comment:
```suggestion
    content_path = CONTENT_PATH
```

wouldn't make more sense?

Reply:
Don't know the full logic behind it, but it's not always the content path.
https://github.com/demisto/demisto-sdk/pull/3917/files#diff-149df266deb904266196e8819d10b53d761b3fb2a4f54ed4003e54727f04adf3R650-R655